README.md

```diff
@@ -1,27 +1,41 @@
 [![MELPA](https://melpa.org/packages/mermaid-mode-badge.svg)](https://melpa.org/#/mermaid-mode)
 
 # mermaid-mode
+
 Emacs major mode for working with [mermaid graphs](https://mermaidjs.github.io/)
 
-![alt text](screenshot.jpg "Screenshot")
+![Screenshot](./assets/screenshot.jpg "Screenshot")
+
+[![Thumbnail](./assets/thumbnail.png)](https://vimeo.com/414458581 "Screencast")
 
 ## Installation
+
 1. Install from Melpa or load the `mermaid-mode.el` file
-2. Install `mmdc` binary from the [mermaid.cli project](https://github.com/mermaidjs/mermaid.cli) if you plan to compile graphs in Emacs
+1. Install `mmdc` binary from the [mermaid.cli project](https://github.com/mermaidjs/mermaid.cli) if you plan to compile graphs in Emacs
```

Comment:
typo?

Reply:
No. I normally let the markdown do the numbering. Then I do not have to re-number everything when I add something in the middle of the list.